ssh/tailssh/tailssh.go

```diff
@@ -197,6 +204,7 @@ type conn struct {
 	idH    string
 	connID string // ID that's shared with control
 
+	anyPasswordIsOkay       bool  // set by NoClientAuthCallback; whether any password is okay for this connection
```

Comment:
is noPubKeyPolicyAuthError no longer used?

Reply:
I didn't touch that?

Reply:
deleted.

Reply:
sent https://github.com/tailscale/tailscale/pull/5898

Reply:
lol that was a race. I'm going to close that PR